docs/content/en/docs/install/install.md

```diff
@@ -80,44 +74,79 @@ as well as for upgrades.
 Some helpful hints:
 
 * Use the `--version <version>` flag on the
-  `helm upgrade --install` command line to specify a different Keptn version.
+  `helm upgrade --install` command to install a Keptn version
+  other than the latest.
+  For example,
+
+  ```shell
+  helm upgrade --install keptn keptn/keptn \
+   --version version klt-v0.8.2 \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+  ```
 
-* Use the following command sequence to see a list of available versions:
+  * Use the following command sequence to see a list of available versions:
+
+    ```shell
+    helm repo update
+    helm search repo keptn
+    ```
+
+* If you modify the `values.yaml` file for the umbrella chart
+  and/or the component sub-charts,
+  apply the modified configuration with the `values - values.yaml` flag.
+  For example, if you created a `my.values.yaml`
+  to exclude the `lifecycle-operator` from your cluster
+  and created a `metrics.values.yaml` to modify the configuration
+  of the `metrics-operator`, the command is:
 
   ```shell
-  helm repo update
-  helm search repo keptn
+  helm upgrade --install keptn keptn/keptn \
+   --values my.values.yaml --values metrics.values.yaml \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
   ```
 
-* To verify that the Keptn components are installed in your cluster,
+  Note that you can specify multiple `--values` options in one command.
+  The precedence is from right to left.
+
+* To view which Keptn components are installed in your cluster
+  and verify that they are the correct ones,
   run the following command:
 
   ```shell
   kubectl get pods -n keptn-lifecycle-toolkit-system
   ```
 
-  The output shows all components that are running on your system.
-
 ## Modify Helm configuration options
 
-Helm chart values can be modified before the installation.
-This is useful if you want to install only the `metrics-operator`
-rather than the full Toolkit
-or if you need to change the size of the installation.
-
-To modify configuration options, download a copy of the
-[chart/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/values.yaml)
-file, modify some values, and use the modified file to install Keptn:
+To modify the Keptn configuration,
+you must modify the appropriate `values.yaml` file.
+Values can be modified before the installation
+or can be modified and reapplied after the initial installation.
+This is useful if you do not want to install all components,
+if you need to modify the configuration in some way,
+such as to turn verbose logging on or off.
```

Comment:
I don't think we need to add any example here. The values files are always the way to configure software that is managed by helm. Any Keptn specific info here will only get outdated at some point.
```suggestion
```

Reply:
I don't understand your comment.  Yes, always modify config using the values.yaml files, but I might need to modify the values.yaml file for metrics-operator or lifecycle-operator, say to use a different port for something or to change the resource limitations.  Now that I'm lookng at the values.yaml files, I don't actually see what you turn verbose logging on but I thought that was in some values.yaml file.  But are you saying that modifying the parameters of the componet subcharts is not something people should do?  If it's something they can do, shouldn't we give some examples of configuration changes that are someone common?

And yes, the documenation will need constant maintenance.  We're trying to minimize the effort here by avoiding redundancy but nothing we write is guaranteed to be true in perpetuity, is it?